styles/learnxinyminutes/catppuccin.user.css

```
@@ -0,0 +1,177 @@
+==/UserStyle== */
+
+@-moz-document domain("learnxinyminutes.com") {
+  @import url("https://unpkg.com/@catppuccin/highlightjs@0.2.2/css/catppuccin.variables.important.css");
+
+  html.light {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  html.dark {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[ @@lookup][ @rosewater];
+    @flamingo: @catppuccin[ @@lookup][ @flamingo];
+    @pink: @catppuccin[ @@lookup][ @pink];
+    @mauve: @catppuccin[ @@lookup][ @mauve];
+    @red: @catppuccin[ @@lookup][ @red];
+    @maroon: @catppuccin[ @@lookup][ @maroon];
+    @peach: @catppuccin[ @@lookup][ @peach];
+    @yellow: @catppuccin[ @@lookup][ @yellow];
+    @green: @catppuccin[ @@lookup][ @green];
+    @teal: @catppuccin[ @@lookup][ @teal];
+    @sky: @catppuccin[ @@lookup][ @sky];
+    @sapphire: @catppuccin[ @@lookup][ @sapphire];
+    @blue: @catppuccin[ @@lookup][ @blue];
+    @lavender: @catppuccin[ @@lookup][ @lavender];
+    @text: @catppuccin[ @@lookup][ @text];
+    @subtext1: @catppuccin[ @@lookup][ @subtext1];
+    @subtext0: @catppuccin[ @@lookup][ @subtext0];
+    @overlay2: @catppuccin[ @@lookup][ @overlay2];
+    @overlay1: @catppuccin[ @@lookup][ @overlay1];
+    @overlay0: @catppuccin[ @@lookup][ @overlay0];
+    @surface2: @catppuccin[ @@lookup][ @surface2];
+    @surface1: @catppuccin[ @@lookup][ @surface1];
+    @surface0: @catppuccin[ @@lookup][ @surface0];
+    @base: @catppuccin[ @@lookup][ @base];
+    @mantle: @catppuccin[ @@lookup][ @mantle];
+    @crust: @catppuccin[ @@lookup][ @crust];
+    @accent-color: @catppuccin[ @@lookup][ @@accent];
+
+    --ctp-rosewater: @rosewater;
+    --ctp-flamingo: @flamingo;
+    --ctp-pink: @pink;
+    --ctp-mauve: @mauve;
+    --ctp-red: @red;
+    --ctp-maroon: @maroon;
+    --ctp-peach: @peach;
+    --ctp-yellow: @yellow;
+    --ctp-green: @green;
+    --ctp-teal: @teal;
+    --ctp-sky: @sky;
+    --ctp-sapphire: @sapphire;
+    --ctp-blue: @blue;
+    --ctp-lavender: @lavender;
+    --ctp-text: @text;
+    --ctp-subtext1: @subtext1;
+    --ctp-subtext0: @subtext0;
+    --ctp-overlay2: @overlay2;
+    --ctp-overlay1: @overlay1;
+    --ctp-overlay0: @overlay0;
+    --ctp-surface2: @surface2;
+    --ctp-surface1: @surface1;
+    --ctp-surface0: @surface0;
+    --ctp-base: @base;
+    --ctp-mantle: @mantle;
+    --ctp-crust: @crust;
+
+    color-scheme: if(@lookup =latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    .share {
+      background-color: @mantle;
+      border-color: @crust;
+    }
+
+    .share:hover {
+      background-color: lighten(@mantle, 10%);
+    }
```

```suggestion
    .share {
      background-color: @mantle;
      border-color: @crust;

      &:hover {
        background-color: lighten(@mantle, 10%);
      }
    }
```

Let's make sure to nest styles, like this. See https://lesscss.org/features/#parent-selectors-feature.

---

Also, let's not use `lighten()` on a core color like mantle. Just use base or surface0, the next two shades.